opae.spec.fedora

```diff
@@ -144,7 +145,7 @@ cp samples/n5010-test/n5010-test.c %{buildroot}%{_usr}/src/opae/samples/n5010-te
 
 
 %if 0%{?rhel}
-  %make_install
+  %cmake_install
```

Comment:
This was per Tom.  Why are we changing it?

Reply:
changes from  %make_build  to %cmake_build because of genearte rpm fails on RHEL 9.0 version